buildSrc/src/main/groovy/org.apache.fineract.release.gradle

```diff
@@ -0,0 +1,266 @@
+                to: "${findProperty('fineract.config.email')}",
+                // to: 'dev@fineract.apache.org',
+                mime: 'text/plain',
+                subjectTemplate: [
+                        templateFile: "${projectDir}/buildSrc/src/main/resources/email/release.step03.branch.subject.ftl"
+                ],
+                messageTemplate: [
+                        templateFile: "${projectDir}/buildSrc/src/main/resources/email/release.step03.branch.message.ftl"
+                ]
+            ]
+        ],
+        step4: [
+            order: 4,
+            description: 'Freeze JIRA version',
+        ],
+        step5: [
+            order: 5,
+            description: 'Create release tag',
+            git: [
+                messageTemplate: [
+                    templateFile: "${projectDir}/buildSrc/src/main/resources/git/release.step05.tag.message.ftl"
+                ]
+            ]
+        ],
+        step6: [
+            order: 6,
+            description: 'Create a distribution',
+        ],
+        step7: [
+            order: 7,
+            description: 'Sign the distribution artifacts',
+            gpg: [
+                file: "${rootDir}/fineract-war/build/distributions/apache-fineract-binary-${project.version}.tar.gz"
+            ]
+        ],
+        step8: [
+            order: 8,
+            description: 'Upload distribution artifacts to staging environment',
+            subversion: [
+                url: 'https://dist.apache.org/repos/dist/dev/fineract',
+                directory: "${System.getProperty("java.io.tmpdir")}/fineract-dist-dev"
+            ],
+        ],
+        step9: [
+            order: 9,
+            description: 'Verify distribution in staging environment',
+        ],
+        step10: [
+            order: 10,
+            description: 'Start voting on the dev mailing list',
+            email: [
+                from: "${findProperty('fineract.config.email')}",
+                name: "${findProperty('fineract.config.name')}",
+                to: "${findProperty('fineract.config.email')}",
+                // to: 'dev@fineract.apache.org',
+                mime: 'text/plain',
+                subjectTemplate: [
+                    templateFile: "${projectDir}/buildSrc/src/main/resources/email/release.step10.vote.subject.ftl"
+                ],
+                messageTemplate: [
+                    templateFile: "${projectDir}/buildSrc/src/main/resources/email/release.step10.vote.message.ftl"
+                ]
+            ]
+        ],
+        step11: [
+            order: 11,
+            description: 'Conclude the vote and announce results',
+            email: [
+                from: "${findProperty('fineract.config.email')}",
+                name: "${findProperty('fineract.config.name')}",
+                to: "${findProperty('fineract.config.email')}",
+                // to: 'dev@fineract.apache.org',
```

Comment:
To avoid any accidental spam I've set the recipient address to the sender. Will modify these values manually during the upcoming release. Once that's all passing I'll make the dev mailing list permanent.